src/app/ApplicationTemplate.Shared/Framework/Startup/StartupBase.cs

```diff
@@ -44,6 +45,10 @@ public void PreInitialize()
 				throw new InvalidOperationException($"You shouldn't call {nameof(PreInitialize)} more than once.");
 			}
 
+#if __ANDROID__ || __IOS__
+			FeatureConfiguration.Style.ConfigureNativeFrameNavigation();
```

Comment:
this code is specific to the application, so it should be in the `Startup` class, not `StatupBase`. It appears like it's missing an abstract method for pre-initialization. You should add it (in the same fashion as `CoreStartupBase.PreInitializeServices`).

Reply:
💯 for the startup did not check properly the app Lifecycle.